integrations/docker/images/stage-2/chip-build-realtek/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG VERSION=1
+FROM ghcr.io/project-chip/chip-build:${VERSION}
+LABEL org.opencontainers.image.source https://github.com/project-chip/connectedhomeip
+
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --no-install-recommends \
+    bzip2 \
+    && rm -rf /var/lib/apt/lists/ \
+    && : # last line
+
+# Setup Realtek
+ARG REALTEK_DIR=/opt/realtek
+
+RUN set -x \
+    && apt-get update \
```

Comment:
Most of the time docker file purpose is to add additional SDKs. If realtek does not need separate SDKs, we should use chip-build directly